### PR TITLE
Fix ReadMe

### PR DESCRIPTION
**Pull request by david-buderus**
### Reason for this PR
- ReadMe was way too old

### Changes in this PR
- Update ReadMe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,96 @@
...
+This file will be load when postgres starts.
+
+## Dependecies of the different services
+![Dependency tree](documentation/dependency_tree.jpg)
+
+The figure gives an overview of the interal communciation between services and therefore there dependencies among themselves. Inter service communcation is either realized through service calls or publish-subscribe techniques using Apache Kafka as message broker. Service calls are synchronous HTTP calls that use the defined REST api, while publishing messages allows asynchronous decoupled communication between services. Messages are published to some topics to which multiple services can subscribe to. Some services communicate with the underlying Hyperledger Fabric Framework, these services are marked with a red tag.
+
+## Running tests in IntelliJ
+IntelliJ expects some impressive Hardware.
+To work around that fact you can tell it to suffice on less Memory.
```
Comment (RHelmert) [inline README.md:83]:
```suggestion
To work around that fact you can tell it to use less Memory.
```

**Review by RHelmert — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,96 @@
...
+Add in .sbtopts '-J' in front of every parameter.
+Should look like this:
+```
+-J-Xms512M
+-J-Xmx6144M
+-J-Xss2M
+-J-XX:MaxMetaspaceSize=2048M
+```
+
+## Run
```
Comment (lrschaef) [inline README.md:39]:
```suggestion
## Running in development mode
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,96 @@
...
+  * [Run](##_Run)
+    * [Errors](###-Errors)
+  * [Change settings of postgres](##-Change-settings-of-postgres)
+  * [Dependecies of the different services](##-Dependecies-of-the-different-services)
+  * [Running tests in IntelliJ](##-Running-tests-in-IntelliJ)
+  * [License](##_License)
+
+---
+
+## Different Versions
```
Comment (lrschaef) [inline README.md:21]:
```suggestion
## Versions in this project
```
Sounds better I think, not a necessary change though

File: README.md [hunk 4]
```diff
@@ -1,6 +1,96 @@
...
+If you encounter any unexpected errors, use following command:
+````shell script
+sbt clean
+````
+
+If this doesn\'t help: Open an issue.
+
+## Changelog
+
+To get an overview of our developmental process, we tagged our releases and added mutltiple changelogs, like the one of the [user service](product_code/user_service/CHANGELOG.md), to our repository which reveals our different releases along with a respective description/enumeration of our changes.
```
Comment (lrschaef) [inline README.md:63]:
```suggestion
To get an overview of our development process, we tagged our releases and added changelogs for every service, like the one of the [user service](product_code/user_service/CHANGELOG.md), to our repository, which lists our different releases along with a description/enumeration of the changes between versions.
```

File: README.md [hunk 5]
```diff
@@ -1,6 +1,96 @@
...
+
+## Change settings of postgres
+The configuration file used by the local docker image
+are defined in the [postgres.conf](product_code/docker/postgres.conf).
+This file will be load when postgres starts.
+
+## Dependecies of the different services
+![Dependency tree](documentation/dependency_tree.jpg)
+
+The figure gives an overview of the internal communciation between services and therefore their dependencies. Inter-service communcation is either realized through service calls or publish-subscribe techniques using Apache Kafka as message broker. Service calls are synchronous HTTP calls that use the defined REST api, while publishing messages allows asynchronous decoupled communication between services. Messages are published to topics which multiple services can subscribe to. Some services communicate with the underlying Hyperledger Fabric Framework. These services are marked with a red tag.
```
Comment (lrschaef) [inline README.md:79]:
```suggestion
The figure gives an overview of the internal communciation between services and therefore their dependencies. Inter-service communcation is either realized through service calls or a publish-subscribe pattern using Apache Kafka as a message broker. Service calls are synchronous HTTP calls that use the defined REST-API and publishing messages to Kafka allows asynchronous decoupled communication between services. Messages are published to topics which multiple services can subscribe to. Some services communicate with the underlying Hyperledger Fabric Framework. These services are marked with a red tag.
```

**Review by lrschaef — APPROVED** [review]

**Pull request merged**